_stbt/ocr.py

```diff
@@ -269,6 +276,11 @@ def ocr(frame=None, region=Region.ALL,
         tesseract_user_patterns, tesseract_user_words, upsample, text_color,
         text_color_threshold, engine, char_whitelist, imglog)
     text = text.strip().translate(_ocr_transtab)
+
+    if corrections is not None:
+        for bad, good in corrections.items():
+            text = re.sub(bad, good, text)
```

Comment:
This will apply corrections to already corrected strings.

Reply:
We could start by sorting the list of keys by length and applying the longest corrections first - excluding things that were replaced from further replacements.

Reply:
The new implementation doesn't re-do the plain-text corrections (the ones that add word boundaries). It does for regexes, but it does them in the order they're specified (similar to nginx location patterns, except that nginx stops at the first match).